packages/optimizers/image/src/lib.rs

```diff
@@ -14,7 +14,6 @@ pub fn optimize(kind: String, buf: Buffer, env: Env) -> Result<JsBuffer> {
   match kind.as_ref() {
     "png" => {
       let options = Options {
-        deflate: Deflaters::Libdeflater,
```

Comment:
This is no longer required as `Libdeflater` is the default: https://github.com/shssoichiro/oxipng/blob/b4e98a4dd16b370a99b31e0125dfa5854fc60dda/src/lib.rs#L313